components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -723,7 +729,9 @@
             if (scim.getEnabled()) {
                 isScimProvEnabled = true;
             }
-
+            if (scimVersion.equals("scim2")) {
```

Comment:
use a constant for "scim2"

Reply:
@emswbandara Please block this PR, since there are valid concerns to fix.

Reply:
Ack